guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
         </mediaobject>
       </screenshot>
     </sect2>
+    
+    <sect2 id="invest-sellLots">
+      <title>Automatic Calculation of Capital Gains or Losses Using Lots</title>
+       
+      <para><application>&app;</application> can automatically calculate and
```

Comment:
I think you need to lead with an explanation of what a "lot" is, then that GnuCash has a built-in lot management facility that can be used to keep track of capital gains.